pandas/io/parquet.py

```diff
@@ -179,6 +180,12 @@ def write(
             mode="wb",
             is_dir=partition_cols is not None,
         )
+
+        # Output compliant Parquet if PyArrow supports it and the user hasn't
+        # explicitly set the desired behavior
```

Comment:
can you add a test which exercises this (you can skip if pa < 5); we almost certainly have a build which will exercise this. wouldn't be against changing *one* build to force pa >= 5

Reply:
I'll see about adding a test but this only needs pa>=4 - not 4?

Reply:
Added a test. I assume it should run on one of the test instances somewhere!

Reply:
ok >4 is fine too